live-build/misc/ansible-roles/appliance-build.virtualization-development/tasks/main.yml

```diff
@@ -49,7 +49,6 @@
     update: no
   with_items:
     - { repo: 'https://gitlab.delphix.com/app/dlpx-app-gate.git',
-        version: projects/dx4linux,
```

Comment:
ditto the comment I left in the masking role; we should remove the `item.version` usage above on linux 47.

Reply:
Sure, done.